src/main/java/TaskManager.java

```diff
@@ -0,0 +1,72 @@
+import java.util.Scanner;
+
+public class TaskManager {
+    private final String BOT_NAME = "[iWish]: ";
+    public Task[] taskList = new Task[100];
+    public int trackList = 0;
+
+    public void startUp() {
```

Comment:
you can split this function into different methods for each case.

Reply:
also, you should try adding error checking, in case the user inputs wrong characters (eg. you are trying to parse an integer but the user does not enter an integer)